### ISSUE TITLE
What everyone using now that this is dead?

### ISSUE THREAD

**Issue by forcergists** [tylertate/semantic.gs#94]
What everyone using now that this is dead?

**Comment by kimslawson** [comment]
@scottstanfield et al., any updates or synthesis of earlier comments? I came here from the Responsable framework as it offered both a horizontal grid (based on semantic.gs) and furthermore provides a vertical baseline grid (also very nice to have). I'd like a LESS/SCSS-agnostic tool (I'm leaning toward SCSS myself but I don't think it should be a deal-breaker either way). Has anyone got the One True Grid (for 2015 ;-) combined with a baseline grid that they like as well?

**Comment by scottstanfield** [comment]
I started a list to look into, but it quickly gets overwhelming. 

I made some progress with http://jeet.gs but I'm not so sure it's the right way to go.

http://roots.cx/
http://tsi.github.io/Salsa/
https://github.com/mojotech/jeet
http://www.profoundgrid.com/
http://laughingwithu.github.io/flawless-semantics-grid/
https://github.com/artofrawr/profoundgrid/
http://framelessgrid.com/
https://github.com/ezekg/flint
https://github.com/RadLikeWhoa/SSGS
http://xycss.com/features/vertical-grid/
http://thesquaregrid.com/#
https://github.com/philipwalton/solved-by-flexbox
jeet.gs
http://www.fluiditycss.com/
https://github.com/aekaplan/grid

I feel like Best kit should have:
- vertical typographic rhythm
- debugging tools
- rem based
- min-width media queries with 3-4 breakpoints
- respects good measure length

Too many choices.

**Comment by scottstanfield** [comment]
One True Grid would be a good name for the 2015 winner.

**Comment by jhli** [comment]
A quick search did not find me One True Grid. Care to link it?

Could you elaborate what you mean by '
respects good measure length'

**Comment by Jogai** [comment]
Here is a nice one: http://webdesign.tutsplus.com/tutorials/a-simple-mixin-alternative-to-standard-css-grids--webdesign-16958

I think it meets the wishes of @scottstanfield   & @ionas 

To cherry pick their points:

* A Grid system that doesn't pollute the HTML. It hides the decisions in CSS, which is where I believe they belong.
* Less syntax
* /just/ a GRID
* GRID that is fully semantic.
* based around LESS

**Comment by ionas** [comment]
I am now using Bootstrap with makeColumn and makeRow. I am extracting just the parts of bootstrap required for my use case. E.g. I don't use anything else from bootstrap as of now.
- Drawback: almost no documentation on this. It is more or less read the source and try to understand etc.
- Benefit: fully tested bootstrap 3 grid behavior e.g. tested across multiple platforms in terms of compatibility
- RAD via LESS.js

Basically what misses is only a good documentation and you'd basically have your semantic less based "grid only fork" of bootstrap.

**Comment by ionas** [comment]
@Jogai http://webdesign.tutsplus.com/tutorials/a-simple-mixin-alternative-to-standard-css-grids--webdesign-16958 also looks quite promising as well!

**Comment by kimslawson** [comment]
Gridle — https://github.com/olivierbossel/gridle — looks promising and is semantic when using mixins.

**Comment by Jogai** [comment]
https://github.com/weepower/wee uses mixins a lot, see their grids:
https://www.weepower.com/style/mixins#grid

Another one: https://github.com/flexiblegs/flexiblegs-scss-plus
